e2e/newenemy/newenemy_test.go

```diff
@@ -270,37 +317,78 @@ func fill(require *require.Assertions, client v0.ACLServiceClient, fillerCount,
 	}
 }
 
-func initSchema(schemaClient v1alpha1.SchemaServiceClient) error {
-	_, err := schemaClient.WriteSchema(context.Background(), &v1alpha1.WriteSchemaRequest{
-		Schema: schema,
-	})
-	return err
+func fillSchema(data []SchemaData, schemaClient v1alpha1.SchemaServiceClient) error {
+	var b strings.Builder
+	batchSize := len(data[0].Prefixes)
+	for i, d := range data {
+		fmt.Println("filling ", i*batchSize, "to", (i+1)*batchSize)
+		b.Reset()
+		err := schemaTpl.Execute(&b, d)
+		if err != nil {
+			return err
+		}
+		_, err = schemaClient.WriteSchema(context.Background(), &v1alpha1.WriteSchemaRequest{
+			Schema: b.String(),
+		})
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// prefixForNode finds a prefix with a leader the node id
+func prefixForNode(ctx context.Context, conn *pgx.Conn, data []SchemaData, node int) (prefix string) {
+	for prefix == "" {
+		// get a random prefix
+		d := data[rand.Intn(len(data))]
+		candidate := d.Prefixes[rand.Intn(len(d.Prefixes))]
+		fmt.Println("checking prefix", candidate)
```

Comment:
Should we be using a logger instead of everywhere in this tool?

Reply:
right now there's a mix of stdout and t.log, so this isn't making things any worse, but yes I think it should be more consistent

Reply:
everything goes through t.Log now